octoprint_v8theme/static/js/v8theme.js

```diff
@@ -1,7 +1,157 @@
 $(function() {
     function V8ThemeViewModel(parameters) {
         var self = this;
-        self.temp = parameters[0]
+        self.temperature = parameters[0];
+        self.terminal = parameters[1];
+        
+        var Colors = ["#fbc08d", "#fde0c8", "#ffb6a6", "#ffdbd4"];
+        self.terminal.lineCount = ko.computed(function() {
+            var total = self.terminal.log().length;
+            var displayed = _.filter(self.terminal.displayedLines(), function(entry) {
+                return entry.type == "line"
```

Comment:
`self.terminal.lineCount` is from OctoPrint source and I just wanted to change some things. Should I go through and edit all of the if statements inside?